config/webpack/docs/webpack.config.dev.js

```diff
@@ -42,6 +42,10 @@ module.exports = {
         test: /\.jsx?$/,
         exclude: [/node_modules/],
         loaders: [require.resolve("babel-loader")]
+      },
+      {
+        test: /\.json$/,
+        loaders: [require.resolve("json-loader")]
```

Comment:
The `json-loader` is part of the dev archetype so needs to use the [module pattern](https://github.com/FormidableLabs/builder#the-module-pattern)

Also, I'm noting deps don't seem to be divided in a consistent sense between `package.json` and `dev/package.json`....